helm/cluster-aws/values.schema.json

```
@@ -76,6 +76,23 @@
                     "type": "integer",
                     "title": "Root volume size (GB)"
                 },
+                "spotInstances": {
+                    "type": "object",
+                    "title": "Spot VM instances",
+                    "description": "Spot VM instances configuration",
```

AWS calls them "spot instances", without the "VM".

Also trying to make the description more useful.

```suggestion
                    "title": "Spot instances",
                    "description": "Compared to on-demand instances, spot instances can help you save cost.",
```

---

Done